src/GRA.Domain.Service/ActivityService.cs

```diff
@@ -1225,14 +1225,14 @@ public async Task<ServiceResult> UpdateFavoriteChallenges(IList<Challenge> chall
                 var notification = new Notification
                 {
                     PointsEarned = 0,
-                    Text = $"<span class=\"fa fa-shopping-bag\"></span> You've unlocked the <strong>{bundle.Name}</strong> avatar bundle!",
+                    Text = $"<span class=\"fa fa-shopping-bag\"></span> You've unlocked the <strong>{bundle.Name}</strong> avatar bundle! You can now see your new bundle on the Avatar's page: <a asp-controller = \"Avatar\" asp-action=\"Index\">Here! </a>",
```

Comment:
Modify this to be:
```
<a asp-controller = \"Avatar\" asp-action=\"Index\">See your new bundle!</a>
```

Reply:
If my Avatar changed worked we probably can simplify this to `Text = $"<span class=\"fa fa-shopping-bag\"></span> You've unlocked the <strong>{bundle.Name}</strong> avatar bundle!",`